doc/project/rfc_process.md

```diff
@@ -1,4 +1,7 @@
-# OpenTitan RFC Process
+---
+aliases: [/doc/project/hw_stages/]
```

Comment:
Did not try to build it locally, but is it intended to have the same alias here as in "development_stages.md"?